types/underscore/underscore-tests.ts

```diff
@@ -152,187 +152,205 @@ declare const maybeFunction: (() => void) | undefined;
 var stooges = [{ name: 'moe', age: 40 }, { name: 'larry', age: 50 }, { name: 'curly', age: 60 }];
 _.pluck(stooges, 'name');
 
-_.max(stooges, (stooge) => stooge.age);
-_.min(stooges, (stooge) => stooge.age);
-_.max({ a: 1, b: 2 });
-_.max({ a: 'a', b: 'bb' }, (v, k) => v.length);
+// retrieving the minimum number in a dictionary
 _.min({ a: 1, b: 2 });
-_.min({ a: 'a', b: 'bb' }, (v, k) => v.length);
 
-var numbers = [10, 5, 100, 2, 1000];
-_.max(numbers);
-_.min(numbers);
+// retrieving the item with the maximum number in a property
+_.max(stooges, (stooge) => stooge.age);
 
+// sorting by a calculated value
 _.sortBy([1, 2, 3, 4, 5, 6], (num) => Math.sin(num));
 
-_([1, 2, 3]).chain()
-    .sortBy(x => -x)
-    .sortBy(x => -x)
-    .value().length;
-
+// grouping numbers by their non-fractional parts
 _([1.3, 2.1, 2.4]).groupBy((e) => Math.floor(e));
-_.groupBy([1.3, 2.1, 2.4], (num) => Math.floor(num).toString());
-_.groupBy(['one', 'two', 'three'], 'length');
-
-_.indexBy(stooges, 'age')['40'].age;
-_(stooges).indexBy('age')['40'].name;
-_(stooges)
-    .chain()
-    .indexBy('age')
-    .value()['40'].age;
 
-let pensioners: string[] = _.chain(stooges)
-    .filter(p => p.age >= 60)
-    .map(p => p.name)
-    .value();
-
-var usersData: _.Dictionary<{ age: number; name: string }> = {
-    'user id': { name: 'moe', age: 40 },
-    'other user Id': { name: 'larry', age: 50 },
-    'fake id': { name: 'curly', age: 60 },
-};
-
-let youngPeopleId: string[] = _.chain(usersData)
-    .map((p, k: string) => k)
-    .value();
-
-let usersTable: { age: number; name: string; id: string }[] = _.chain(usersData)
-    .map((p, k: string) => {
-        return { id: k, ...p };
-    })
-    .value();
-
-// Test map function with _ChainOfArrays<>
-let usersTable_2 /*: { age: number; name: string; id: string }[][]*/ = _.chain(usersData)
-    .map((p, k: string) => {
-        return [{ id: k, ...p }];
-    })
-    .value();
+// grouping numbers by the value of a specified property
+_.groupBy(['one', 'two', 'three'], 'length');
 
-let usersTable_3 /*: { score: number; fullName: string; login: string }[][]*/ = _.chain(usersTable)
-    .map(p => {
-        return [
-            {
-                login: p.id,
-                fullName: p.name,
-                score: p.age,
-            },
-        ];
-    })
-    .value();
+// indexing items in a dictionary by age
+_.indexBy(stooges, 'age');
 
+// counting numbers by their evenness
 _.countBy([1, 2, 3, 4, 5], (num) => (num % 2 == 0) ? 'even' : 'odd');
 
+// shuffling numbers
 _.shuffle([1, 2, 3, 4, 5, 6]);
 
+// converting an array-like structure to an array
 (function (a, b, c, d) { return _.toArray(arguments).slice(1); })(1, 2, 3, 4);
 
+// determining the number of items in a dictionary
 _.size({ one: 1, two: 2, three: 3 });
 
+// splitting numbers into sets of even and odd values
 _.partition<number[]>([0, 1, 2, 3, 4, 5], (num) => {return num % 2 == 0 });
```

Comment:
Is that type annotation still necessary?

Reply:
Nope! The end state does have that removed, but my plan in this subset was to avoid changes that seemed like they might fit better in subset A to keep the diff more readable. I'm happy to remove it now, though 😄

Reply:
Removed this.